packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -273,7 +273,7 @@ contract Deploy is Deployer {
     /// @notice Internal function containing the deploy logic.
     function _run() internal {
         console.log("start of L1 Deploy!");
-        deploySafe();
+        deploySafe("SystemOwnerSafe");
```

Comment:
I think it's to avoid renaming for now, but perhaps worth adding a comment explaining what [role](https://docs.optimism.io/chain/security/privileged-roles) it's actually referring to

Reply:
Well, it doesn't even have a lasting role, IIRC it transfers ownership to the `finalSystemOwner` at the end. This safe is basically thrown away.